FLEDGE.md

```diff
@@ -492,15 +492,21 @@ For the JSON response, only the `https` scheme is supported -- the `uuid-in-pack
 
 #### 2.5.2 Using Response Headers
 
-An alternative way to pass DirectFromSellerSignals without subresource bundles is via the `Ad-Auction-Signals` response header of some `fetch()` request, together with the `directFromSellerSignalsHeaderAdSlot` parameter on `navigator.runAdAuction()`.
+An alternative way to pass DirectFromSellerSignals without subresource bundles is via the `Ad-Auction-Signals` response header of some `fetch()` request or `iframe` navigation, together with the `directFromSellerSignalsHeaderAdSlot` parameter on `navigator.runAdAuction()`.
```

Comment:
I assume we haven't implemented this yet, so we don't need to update WPT yet?

Reply:
I just submitted the implementation. Updating WPT is on my TODO list.